packages/contracts/contracts/L1/AssetManager.sol

```diff
@@ -1022,24 +797,26 @@ contract AssetManager is ISemver, IERC721Receiver, IAssetManager {
      * @param kghCount  The number of KGHs to delegate.
      * @param kroInKghs The amount of KRO in the KGHs.
      * @param kroShares The amount of KRO shares to receive.
-     * @param kghShares The amount of KGH shares to receive.
      */
     function _delegateKghBatch(
```

Comment:
Please update the comment

Reply:
I wonder why didn't you unify the input and action between this function and `_delegateKgh`.